frontend/src/lib/utils.test.ts

```diff
@@ -752,6 +753,122 @@ describe('lib/utils', () => {
         })
     })
 
+    describe('autoCaptureEventToDescription()', () => {
```

Comment:
Most of those are similar to the `eventToDescription` one for non-autocaptured events but the setup is different enough that I guess the redundancy was ok for tests.